app/models/stolen_record.rb

```diff
@@ -100,7 +109,15 @@ def self.locking_defeat_description_select
     locking_defeat_description.map { |l| [l, l] }
   end
 
-  before_save :set_phone, :fix_date, :titleize_city, :update_tsved_at
+  before_save :set_calculated_attributes
```

Comment:
The new approach I'm taking with putting all callbacks (ie `before_save ...`) up at the top, after the scopes. It makes it easier to find. You don't need to move the method, but I would like it if you moved this up there to the top!